examples/mongodb.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+require '../vendor/autoload.php';
+
+use Doctrine\MongoDB\Connection;
+use Doctrine\ODM\MongoDB\Configuration;
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Doctrine\ODM\MongoDB\Mapping\Driver\AnnotationDriver;
+
+use Conticket\Document;
+
+AnnotationDriver::registerAnnotationClasses();
+
+$config = new Configuration();
+$config->setProxyDir('../data/cache/proxy');
+$config->setProxyNamespace('Proxies');
+$config->setHydratorDir('../data/cache/hydrators');
+$config->setHydratorNamespace('Hydrators');
+$config->setMetadataDriverImpl(AnnotationDriver::create('../data/cache/classes'));
```

Comment:
If you are using this directory to cache, you should commit this strutctur `data/cache/classes` with a `.gitkeep` empty file to preserve the structure